libevm/VMOpt.cpp

```diff
@@ -119,9 +126,10 @@ void VM::optimize()
 		}
 #endif
 	}
-	
-	copyCode(pc - nBytes + 33);
```

Comment:
Somehow when I merged this code a line got left behind.  At VMOpt:125, just inside the BEGINDATA case, it should say
> pc = nBytes;

I think the implication is that this loop cannot in fact make the data bigger, I just thought once that the BEGINDATA payload might.